e-commerce-de.md

```diff
@@ -0,0 +1,18 @@
+# E-commerce Website
+
+## Willkommen in unserem Geschäft! 🛒😃
+
+### Produktkategorien.
+
+- [ ] Elektronik 📱💻🎮
```

Comment:
Hey, the Elektronik should be toggled

Reply:
I fixed this in this commit: c3d4d36fd197d891a87421feb322b4d6fc217132